docs/API.md

```diff
@@ -2,46 +2,56 @@
 
 ### gulp.src(globs[, options])
 
-Emits files matching provided glob or an array of globs. 
-Returns a [stream](http://nodejs.org/api/stream.html) of [Vinyl files](https://github.com/wearefractal/vinyl-fs) 
-that can be [piped](http://nodejs.org/api/stream.html#stream_readable_pipe_destination_options) 
-to plugins.
+Emits files matching provided glob or an array of globs.
+Returns a [stream] of [Vinyl files] [piped] to plugins.
 
-```javascript
+```js
 gulp.src('client/templates/*.jade')
   .pipe(jade())
   .pipe(minify())
   .pipe(gulp.dest('build/minified_templates'));
 ```
 
-`glob` refers to [node-glob syntax](https://github.com/isaacs/node-glob) or it can be a direct file path.
+`glob` refers to [node-glob syntax][node-glob] or it can be a direct file path.
 
 #### globs
 Type: `String` or `Array`
 
 Glob or array of globs to read.
 
+Note that globs are evaluated in order, which means this is possible:
+```
+// exclude every JS file that starts with a b except bad.js
+gulp.src(['*.js', '!b*.js', 'bad.js'])
+```
+
 #### options
 Type: `Object`
 
 Options to pass to [node-glob] through [glob-stream].
 
-gulp adds some additional options in addition to the [options supported by node-glob][node-glob documentation] and [glob-stream]:
+gulp adds some additional options in addition to the
+[options supported by node-glob][node-glob documentation] and [glob-stream]:
```

Comment:
It's a bit odd, imo, especially here:

https://github.com/gulpjs/gulp/pull/937/files#diff-96cc378dcdef180641176501f42183c7R286

Imo (and what I usually do) is either consistently break the lines all at 80 characters, or don't break them.